aspnet/mvc/views/view-components.rst

```diff
@@ -77,184 +120,116 @@ Notes on the code:
-
-Examine the view component view
--------------------------------
-
-1. Examine the contents of the *Views\\Todo\\Components*. This folder **must** be named *Components*.
+- The component uses `dependency injection <https://docs.asp.net/en/latest/fundamentals/dependency-injection.html>`__ to make the data context available. 
+- ``InvokeAsync`` exposes a method which can be called from a view, and it can take an arbitrary number of arguments. 
+- The ``InvokeAsync`` method returns the set of ``ToDo`` items that are not completed and have priority lower than or equal to ``maxPriority``.
 
-.. note:: View Component views are more typically added to the *Views\\Shared\\Components* folder, because view components are typically not controller specific.
+Create the view component Razor view
+--------------------------------------
 
-2. Examine the *Views\\Todo\\Components\\PriorityList* folder. This folder name must match the name of the view component (name of the class minus the suffix if we followed convention and used the *ViewComponent* suffix in the class name). If you used the the ``ViewComponent`` attribute, the folder name would need to match the attribute designation. 
+#. Create the *Views/Shared/Components* folder. This folder **must** be named *Components*.
+#. Create the *Views/Shared/Components/PriorityList* folder. This folder name must match the name of the view component class, or the name of the class minus the suffix (if we followed convention and used the *ViewComponent* suffix in the class name). If you used the ``ViewComponent`` attribute, the class name would need to match the attribute designation.
+#. Create a *Views/Shared/Components/PriorityList/Default.cshtml* Razor view. 
 
-3. Examine the *Views\\Todo\\Components\\PriorityList\\Default.cshtml* Razor view. 
+.. literalinclude:: view-components/sample/ViewCompFinal/Views/Shared/Components/PriorityList/Default1.cshtml
+ :language: html 
 
-  .. code-block:: html
-    :linenos:
-    
-    @model IEnumerable<TodoList.Models.TodoItem>
-
-    <h3>Priority Items</h3>
-    <ul>
-      @foreach (var todo in Model)
-      {
-        <li>@todo.Title</li>
-      }
-    </ul>
+The Razor view takes a list of ``TodoItem`` and displays them. If the view component ``InvokeAsync`` method doesn't pass the name of the view (as in our sample),  *Default* is used for the view name by convention. Later in the tutorial, I'll show you how to pass the name of the view. If the view component was controller specific, you could add it to the controller specific folder (*Views/Todo/Components/PriorityList/Default.cshtml*).
```

Comment:
@rynowak How's this
To override the default styling for a specific controller, add a view to the controller specific view folder (for example Views/Todo/Components/PriorityList/Default.cshtml).